assets/stylesheets/shared/_color-schemes.scss

```
@@ -175,7 +175,11 @@
 	--color-error-800-rgb: #{hex-to-rgb( $muriel-hot-red-800 )};
 	--color-error-900: #{$muriel-hot-red-900};
 	--color-error-900-rgb: #{hex-to-rgb( $muriel-hot-red-900 )};
-
+	
+	--color-orange: #{$muriel-hot-orange-500};	
```

if we're going to go with straight color names, I think we'd want to keep the `hot` bit here. 

We've thought about exposing all of the `muriel` colors as color variables... Maybe a separate PR?

---

Me doesn't follow. :sweat_smile:

> if we're going to go with straight color names, I think we'd want to keep the hot bit here.

So... 

```suggestion
	--color-hot-orange: #{$muriel-hot-orange-500};	
```

> We've thought about exposing all of the muriel colors as color variables... Maybe a separate PR?

As in, all the colours from Colour Studio as colour variables, 0-900 (rgb too)?

---

Your suggestion look good to me 👍 

> As in, all the colours from Colour Studio as colour variables, 0-900 (rgb too)?

Yes, though it's not a blocker for this PR and could be done in a separate one.

---

Oh, okay. :+1: